server/etcdserver/txn/txn.go

```diff
@@ -138,6 +143,13 @@ func Range(ctx context.Context, lg *zap.Logger, kv mvcc.KV, r *pb.RangeRequest)
 		trace = traceutil.New("range", lg)
 		ctx = context.WithValue(ctx, traceutil.TraceKey{}, trace)
 	}
+	op := "range"
+	stringer := &pb.InternalRaftStringer{Request: &pb.InternalRaftRequest{Range: r}}
+	defer func(start time.Time) {
+		success := err == nil
+		RangeSecObserve(v3Version, op, success, time.Since(start))
```

Comment:
I think it's better to remove "version" label, because it's always same value. The only benefit is that it follows `applySec` pattern.